homework/shelkovich_ilya/primitiveAndWrapperSpeed/src/test/java/com/tutrit/java/quickstart/service/SumTest.java

```diff
@@ -0,0 +1,21 @@
+package com.tutrit.java.quickstart.service;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SumTest {
+
+    Sum sum;
+
+    @Before
+    public void setUp() {
+        this.sum = new Sum();
+    }
+
+    @Test
+    public void timeToCount(){
+        Assert.assertTrue(sum.millsToCalculatePrimitives() < sum.millsToCalculateWrappers());
```

Comment:
If you change it to 
`Assert.assertTrue(sum.millsToCalculateWrappers() < sum.millsToCalculatePrimitives());`
it would still pass. 

See comment above and let's discuss why if happens.

Reply:
fixed program logic, now it works